### DISCUSSION TITLE
Wrong link in the DockerHub page

### DISCUSSION THREAD

**Discussion by j178** [GreptimeTeam/greptimedb#2818] [General]
The link to GitHub in this DockerHub page https://hub.docker.com/r/greptime/greptimedb is wrong:

![image](https://github.com/GreptimeTeam/greptimedb/assets/10510431/990fa2e7-7483-466f-9a67-113923fb80e4)

**Comment by zyy17** [comment]
@j178 Thank you for your report! We have already fixed the typo. Please have a try 😊

**Reply by j178** [reply]
Yes, I see it was changed, but still wrong there 😳
<img width="570" alt="image" src="https://github.com/GreptimeTeam/greptimedb/assets/10510431/47400d25-d9a9-4bcb-b0d8-303fce7d9586">

**Reply by zyy17** [reply]
ops...fix again.
